### PR TITLE
Make test DRYer

### LINKED ISSUE #4716: Random BDD test failure
Having opened a lot of PRs (25 in a day), I found that I had to run the same tests several times because of this BDD test failure:

```
  Scenario: Content is available        # kalite/distributed/features/learn_content.feature:9
    Given I open some available content # kalite/distributed/features/steps/learn_content.py:17
    Given I open some available content # kalite/distributed/features/steps/learn_content.py:17 4.449s
    Then I should see no alert          # kalite/distributed/features/steps/learn_content.py:33
    Then I should see no alert          # kalite/distributed/features/steps/learn_content.py:33 1.732s
      Assertion Failed: NoSuchElementException not raised
```

It occurs alone. Often after dark. It's very hard to spot because it's very camouflaged. Only a vague red stripe distinguishes it.

IMO we should get it fixed before releasing because it's going to be annoying for both this release and any future fixes.

@aronasorman please re-assign at will :)

### PR DESCRIPTION
**Pull request by MCGallaspy**
Don't write the same code to rate an element twice. Just re-use.

Fixes #4716 again.

### CODE REVIEW

File: kalite/distributed/features/steps/content_rating.py [hunk 1]
```diff
@@ -151,14 +162,15 @@ def impl(context):
...
+
+def enter_star_ratings(context, val=3):
     """
     Enters a value for all three star rating forms, on a new form
     :param context: behave context
     :return: nothing
     """
     for id_ in STAR_CONTAINER_IDS:
-        rate_id(context, id_)
+        rate_id(context, id_, val=val)
```
Comment (benjaoming) [inline kalite/distributed/features/steps/content_rating.py]: That seems critical!!

**Comment by benjaoming** [comment]
:+1:

**Comment by benjaoming** [comment]
Just waiting for Circle...

**Comment by MCGallaspy** [comment]
Something's up. Passes locally. Increasing timeout, will reconsider if that doesn't work.

**Comment by benjaoming** [comment]
Well, no consolidations, but mine also passed a lot locally before Circle failed :)

Btw in case you didnt know.... `bin/kalite manage test distributed --bdd-only --failfast`

**Comment by MCGallaspy** [comment]
AFAICT this is not a legitimate bug. Screencast seems to indicate that not everything had finished loading before the timeout was reached, otherwise the sidebar would have collapsed.

**Comment by benjaoming** [comment]
@MCGallaspy yup, that's how all the failing screen casts I've looked at during this BDD debugging

**Pull request merged**